routing/pathfind_test.go

```diff
@@ -2224,6 +2224,52 @@ func TestNoCycle(t *testing.T) {
 	}
 }
 
+// TestRouteToSelf tests that it is possible to find a route to the self node.
+func TestRouteToSelf(t *testing.T) {
+	t.Parallel()
+
+	testChannels := []*testChannel{
+		symmetricTestChannel("source", "target", 100000, &testChannelPolicy{
+			Expiry: 144,
+		}, 1),
+	}
+
+	ctx := newPathFindingTestContext(t, testChannels, "source")
+	defer ctx.cleanup()
+
+	const (
+		startingHeight = 100
+		finalHopCLTV   = 1
+	)
+
+	paymentAmt := lnwire.NewMSatFromSatoshis(100)
+	target := ctx.source
+
+	// Find the best path given the restriction to only use channel 2 as the
+	// outgoing channel.
```

Comment:
Nit: this comment doesn't seem to apply here.

Reply:
Fixed